### ISSUE TITLE
Support OwnCloud, NextCloud

### ISSUE THREAD

**Issue by marathone** [mrgodhani/raven-reader#285]
Be quite nice if RavenReader could support NextCloud. I'm assuming it doesn't now as it's not explicitly mentioned that I can see.
Thank-you

**Comment by mrgodhani** [comment]
@marathone It doesn't. But will look into it. Again bear with me it's one developer myself so new features take a bit time.

**Comment by marathone** [comment]
Thanks @mrgodhani NextCloud has an API for fetching/reading/syncing. So shouldn't be too much work. Thanks again.

**Comment by TobiasDev** [comment]
Hello! Just short question, are there any updates for this? I don't mean to stress, just curios if this is on the roadmap? :)

**Comment by mrgodhani** [comment]
@TobiasDev Yes, it is in pipeline sorry it's only one man working so might take time.

**Comment by ei8fdb** [comment]
Hi there, I know this ticket is closed, but I'd like to ask if the request for supporting Nextcloud is still in the pipeline? Has it moved closer to the end? At the same location? Just asking. Thanks. :+1:

**Comment by HammyHavoc** [comment]
I'd like to see this happen too. Could be the best RSS reader if this was possible.

**Comment by MyNameIsTroll** [comment]
I discover this great application, it is true that it would be nice to be able to synchronize its subscriptions with Nextcloud.

**Comment by JoshuaPettus** [comment]
Oh sure I hope you can!  One look and I can tell this is a really nice project.  Such excellent work!  Unfortunately, this is the one feature I would need to move from RSS Guard.  The Nextcloud News App (NC's largest and most supported RSS aggregator app) does have it's own API you can use.   That would be the program you would be interfacing and not NC directly.
